samples/subsys/tracing/README.rst

```diff
@@ -0,0 +1,87 @@
+.. _tracing_sample:
+
+Send Tracing Formatted Packet To The Host With Supported Backends
+#################################################################
+
+Overview
+********
+
+This application can be used to demonstrate the tracing feature. The tracing
+formatted packet will be sent to the host with the currently supported tracing
+backend under tracing generic infrastructure.
+
+Requirements
+************
+
+Depends of the boards which you are using, choose one of .conf files for use tracing subsys.
+
+Usage for UART Tracing Backend
+******************************
+
+Build a UART-tracing image with:
+
+.. code-block:: console
+
+	cmake -DBOARD=mps2_an521 -DCONF_FILE=prj_uart.conf ..
+
+or:
+
+.. code-block:: console
+
+	cmake -DBOARD=mps2_an521 -DCONF_FILE=prj_uart_ctf.conf ..
+
+- NOTE: You may need to set 'zephyr,tracing-uart' property under the chosen
+node in your devicetree.  See :zephyr_file:`boards/mps2_an521.overlay` for an example.
+
+After the application has run for a while, check the trace output file.
+
+
+Usage for USB Tracing Backend
+******************************
+
+Build a USB-tracing image with:
+
+.. code-block:: console
+
+	cmake -DBOARD=sam_e70_xplained -DCONF_FILE=prj_usb.conf ..
```

Comment:
I think it would be great to use the opportunity to switch this one and the following to using west. 
You can have a look at the syntax here https://github.com/zephyrproject-rtos/zephyr/blob/f1c0eb915b13db0b20875aad7bf320497dd6da33/samples/net/sockets/http_get/README.rst?plain=1#L49-L54